data-preprocessing/JSON  Section/main.py

```diff
@@ -0,0 +1,75 @@
+import json
+import os
+import csv
+
+# absolute location of the json files
+PATH = "."
```

Comment:
also don't use this variable name `PATH` means something very specific in our computer systems. Please use a more descriptive name for your use case